src/erlcloud_s3.erl

```diff
@@ -851,7 +863,7 @@ s3_request(Config, Method, Host, Path, Subresource, Params, POSTData, Headers) -
 s3_request(Config, Method, Host, Path, Subresources, Params, POSTData, Headers, GetOptions) ->
     {ContentMD5, ContentType, Body} =
         case POSTData of
```

Comment:
Sorry that it's not directly related this PR, `POSTData` can be `binary()` as well as `{_, _}` because line 181 (in new version) passes `<<>>` as `POSTData`.